initial/python_3

```diff
@@ -101,23 +101,43 @@ from battle import commander
 client = commander.Client()
 
 
-def unit_in_range(data, **kwargs):
+def search_next_target(data, **kwargs):
+    client.unsubscribe_all()
+    enemies = client.ask_enemy_items_in_my_firing_range()
+    if enemies:
+        unit_in_firing_range(enemies[0])
+    else:
+        client.subscribe_enemy_in_my_firing_range(unit_in_firing_range)
+
+
+def unit_in_firing_range(data, **kwargs):
+    print(data)
     client.attack_item(data['id'])
+    client.subscribe_the_item_is_dead(data['id'], search_next_target)
+    client.subscribe_the_item_out_my_firing_range(data['id'], search_next_target)
```

Comment:
this two events can be united in one "on_wait" or "on_done" that will work in the same way as "on_stand" for units when they are required for a new command